launcher/src/preload.js

```diff
@@ -11,4 +11,8 @@ contextBridge.exposeInMainWorld("promiseIpc", {
     ipcRenderer.on("terminal-output", (_, arg) => callback(arg));
     return () => ipcRenderer.removeListener("terminal-output", callback);
   },
+
+  // Audio control functions
+  setVolume: (volume) => ipcRenderer.invoke("set-system-volume", volume),
+  getVolume: () => ipcRenderer.invoke("get-system-volume"),
```

Comment:
this is not right handle it like we always did in `ControlService.js` and don't access 'window.promiseIpc` from frontend.